pkg/status/helpers.go

```diff
@@ -34,18 +33,6 @@ func doNotEscape(value string) template.HTML {
 	return template.HTML(value)
 }
 
-func pythonLoaderError(value string) template.HTML {
```

Comment:
Was this function never used?

Reply:
Since I fixed the traceback formatting we no longer need this (if you remove the formatting of a python traceback the code is the same as `doNotEscape`).